tools/esdlcmd/esdl2ecl.cpp

```diff
@@ -367,15 +374,19 @@ class Esdl2EclCmd : public EsdlConvertCmd
                 "esdl ecl sourcePath outputPath [options]\n"
                 "\nsourcePath must be absolute path to the ESDL Definition file containing the"
                 "EsdlService definition for the service you want to work with.\n"
-                "outputPath must be the absolute path where the ECL output with be created."
+                "outputPath must be the absolute path where the ECL output with be created.\n"
                 "   Options:\n"
                 "      -x, --expandedxml     Output expanded XML files\n"
                 "      --includes            Process all included files\n"
-                "      --rollup              Roll-up all processed includes to single ecl output file"
+                "      --rollup              Roll-up all processed includes to single ecl output file\n"
+                "      --CDE, --compfilesdir HPCC Component files directory (xslt files)\n"
+                "      --ecl-includes        Coma-delimited import list to be attached to output ECL\n"
```

Comment:
changed.

Regarding ECLImport comment, it sounds like a good feature enhancement but somewhat out of the scope of this fix. This change was only intended to removed the hard-coded import $.share statement preventing ESDL ecl filename.ecm to work for all cases. Perhaps a new ticket should be opened for that feature.